content/en/docs/reference/using-api/client-libraries.md

```diff
@@ -77,6 +77,6 @@ their authors, not the Kubernetes team.
 | Ruby                 | [github.com/kontena/k8s-client](https://github.com/kontena/k8s-client) |
 | Rust                 | [github.com/clux/kube-rs](https://github.com/clux/kube-rs) |
 | Rust                 | [github.com/ynqa/kubernetes-rust](https://github.com/ynqa/kubernetes-rust) |
-| Scala                | [github.com/hagay3/skuber](https://github.com/hagay3/skuber) |
+| Scala                | [github.com/doriordan/skuber](https://github.com/doriordan/skuber) |
```

Comment:
Hi @briansterle .
Can you clarify why the `hagay3` Scala client library link should be removed (https://github.com/hagay3/skuber)?
This library is version 2.7.1?

Reply:
I'm thinking the fork (hagay3) was linked in the K8s docs in error. The original Skuber library (with ~300 stars) should be linked, I would think, rather than the fork that has less than 10 stars. 

The fork has been updated more recently, but if these are valuable updates, I would think those will get merged into the original library in the future.

Reply:
@briansterle Heya, given that we try to be as neutral as possible, my suggestion would be that we list both projects, noting that one of them is a fork. Does that sound fine?